CMakeModules/EosioTester.cmake.in

```diff
@@ -102,6 +103,7 @@ macro(add_eosio_test_executable test_name)
        ${libchainbase}
        ${libbuiltins}
        ${libsecp256k1}
+       ${libff}
```

Comment:
also needs GMP I believe

Reply:
should only need one line we previously had

https://github.com/eosnetworkfoundation/mandel/blob/dc1c5c08d6f14e99305352fe6697969fb177044a/CMakeModules/EosioTester.cmake.in#L105